extensions/blocks/anchor-fm/editor.js

```diff
@@ -48,6 +51,26 @@ async function insertSpotifyBadge() {
 	}
 }
 
+const ConvertToAudio = () => (
+	<PluginPostPublishPanel>
+		<p className="post-publish-panel__postpublish-subheader">
+			<strong>{ __( 'Convert to audio', 'jetpack' ) }</strong>
+		</p>
+		<p>{ __( 'Let your readers listen to your post.', 'jetpack' ) }</p>
+		<p>
+			<a href="https://anchor.fm/wordpress" target="_top">
+				{ __( 'Create a podcast episode', 'jetpack' ) }
+			</a>
```

Comment:
Since it leads to an external site, shouldn't this link be using `ExternalLink`?

Reply:
Unfortunately the `ExternalLink` component does not allow to set a custom `target` value and it will open the URL in a new tab: 

https://github.com/WordPress/gutenberg/blob/b52d33bdaf881271ff31705c8d5985d3a55118f5/packages/components/src/external-link/index.js#L34

Users opening this link will be back at the editor once they publish an episode in Anchor, so ideally we would like to open the URL in the same tab to avoid users ending up with the editor opened in 2 tabs.

But I agree that we should denote somehow this will link to an external URL so I added the external link icon to the anchor in c0e9430:
<img width="280" alt="Screen Shot 2020-12-14 at 11 40 05" src="https://user-images.githubusercontent.com/1233880/102071747-33ce9c80-3e01-11eb-9146-14bae4e7fac5.png">

Reply:
👍

My external icon isn't as pretty as yours (WP 5.6, no Gutenberg), but I don't think this is a blocker for now, this can be addressed in a follow-up PR when we'll update the Anchor.fm link.

![image](https://user-images.githubusercontent.com/426388/102074098-89f10f00-3e04-11eb-8496-40643ab93485.png)